spring-pulsar-spring-boot-autoconfigure/src/test/java/org/springframework/pulsar/autoconfigure/PulsarPropertiesTests.java

```diff
@@ -511,7 +511,7 @@ class ReaderPropertiesTests {
 		void readerProperties() {
 			Map<String, String> props = new HashMap<>();
 
-			props.put("spring.pulsar.reader.topic-names", "my-topic");
```

Comment:
Oh, sorry @sobychacko - I should have clarified. I meant that we typically use the relaxed named binding on the "left hand side" as that is typically what peepz will use in yml so it theoretically helps catch that.